android/README.md

```diff
@@ -0,0 +1,50 @@
+<p align="center">
+  <a href="https://github.com/ForgeRock/sdk-sample-apps">
+    <img src="https://cdn.forgerock.com/logo/interim/Logo-PingIdentity-ForgeRock-Hor-FullColor.svg" alt="Logo">
+  </a>
+  <hr/>
+</p>
+
+## Samples
+
+Ping provides these Android samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of Ping and are not officially supported.
+
+### Integrate with PingOne Advanced Identity Cloud / PingAM:
+
+To try out the Ping Android SDK please look at one of our samples:
```

Comment:
Let's write around places where we say "the Ping Android SDK", as I don't think that's an official product name, and might clash with the hundreds of other SDKs we have ;)

Perhaps something like:
The following Android samples are available in this repo:

Reply:
Yeah is it's still up in the air I suggest writing around it, so you don't have to refer to it by a name, there's really no need that often.

So for example here we could just write:

"The following Android samples are available in this repo:"

No need to mention a name at all. Where we do, just "SDK" works, whatever the name becomes.